fe/fe-core/src/main/java/com/starrocks/clone/DynamicPartitionScheduler.java

```diff
@@ -186,6 +187,11 @@ public void createOrUpdateRuntimeInfo(String tableName, String key, String value
                 partitionProperties
                         .put("replication_num", String.valueOf(dynamicPartitionProperty.getReplicationNum()));
             }
+            Map<String, String> tableProperty = olapTable.getTableProperty().getProperties();
+            if (tableProperty != null && tableProperty.containsKey(PropertyAnalyzer.PROPERTIES_STORAGE_MEDIUM)) {
+                partitionProperties.put(PropertyAnalyzer.PROPERTIES_STORAGE_MEDIUM,
+                        tableProperty.get(PropertyAnalyzer.PROPERTIES_STORAGE_MEDIUM));
+            }
```

Comment:
I think it is better to put this into Catalog.getOrSetDefaultProperties when addPartitions.
In this way, both alter table and dynamic partition scheduler add partition are effective.
Only dynamic properties need to be checked here.